Add directions how to find UniFi Protect: nvr_id, camera_id, event_id… (#28342)
home-assistant/home-assistant.github.io: source/_integrations/unifiprotect.markdown

`nvr_id` can either be the UniFi Protect ID of your NVR or the config entry ID for your UniFi Protect integration. `camera_id` can either be the UniFi Protect ID of your camera or an entity ID of any entity provided by the UniFi Protect integration that can be reversed to a UniFi Protect camera (i.e., an entity ID of a detected object sensor).

> The easiest way to find the `nvr_id`, `camera_id`, `start` and `end` times is from viewing one of the videos from UniFi Protect in the Media browser. If you open the video in a new browser tab you will see all of these values in the URL. The `start` time is the last_changed timestamp of the event when the sensor started detecting motion and the `end` time is the last_changed timestamp of the event when the sensor stopped detecting motion. Similarly, the `event_id` for the image can also be found from the event when the sensor starts detecting motion and can be viewed in the developer tools.

Reviewer: ```suggestion
The easiest way to find the `nvr_id`, `camera_id`, `start`, and `end` times is by viewing one of the videos from UniFi Protect in the Media browser. If you open the video in a new browser tab, you will see all these values in the URL. The `start` time is the last_changed timestamp of the event when the sensor started detecting motion. The `end` time is the last_changed timestamp of the event when the sensor stopped detecting motion. Similarly, to see the `event_id` of the image, go to {% my developer_states title="**Developer Tools** > **States**" %} and find the event when the sensor started detecting motion.
```
I'm not entirely sure about the last sentence. But that is how I understood it. Please correct me if I got this wrong.

Author: The last sentence isn't correct. You need to go to Developer Tools > States. Is this correct markdown? 

`go to {% my developer_events title="**Developer Tools** > **States**" %} and filter on the entity where you expect to see the motion detection.`

I can't see this templating stuff in the preview so I wasn't able to verify.

Reviewer: Thank you for checking! Fixed it. Yes, the My link is fine.
https://my.home-assistant.io/faq/
